pkgs/development/python-modules/sqlalchemy-adapter/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3,
```

```suggestion
  buildPythonPackage,
```

And others. Don't reference python3 here.